### ISSUE TITLE
Agency Address Implementation needs help

### ISSUE THREAD

**Issue by trystant** [BOWiki/BOW#679]
Most agencies occupy more than one building and have multiple addresses, so unless we plan on listing all the addresses & buildings owned by an agency, it doesn't make sense to have a single address for it. I would take out the address from the current agency implementation and think about how to implement this.

**Comment by campgurus** [comment]
True, but agencies also generally have a single headquarters.  The address provides a point of contact for people who want to get in touch with the agency. Also, though we currently validate for name uniqueness, it seems somewhat awkward to have to name the Columbus Ohio Police Department and the Columbus Georgia Police Department.  To the extent we find a solution that doesn't require the validation, the address helps friendly_id generate unique urls.

**Comment by trystant** [comment]
The headquarters assertion makes sense, but why not validate on name and location? That way you wouldn't have to worry about " Columbus Ohio Police Department" and "Columbus Georgia Police Department".

BTW, the official names of those agencies are:

* [Division Of Police, Columbus Ohio](https://www.columbus.gov/police/) and
* [Police Department of Columbus, Georgia](http://www.columbusga.org/police/)

So if we're using their proper names, that conflict wouldn't exist in any case.

**Comment by rlgreen91** [comment]
I agree, we should validate on name and location.  And I think we should push people to use official names.  If possible, this is something we would ask supervising editors to investigate and verify.

We will need to go through a process to make the location change, yes?  Unless the address field was removed and this issue wasn't closed...
